crates/bevy_color/src/color.rs

```diff
@@ -1,8 +1,8 @@
 use crate::{
     Alpha, Hsla, Hsva, Hwba, Laba, Lcha, LinearRgba, Oklaba, Oklcha, Srgba, StandardColor, Xyza,
 };
+#[cfg(feature = "serialize")]
```

Comment:
The cfg is not needed here, rather, it'll cause problems if the feature is off (as we reflect some types here)
```suggestion
```

Reply:
Probably removing this cfg above reflection imports on the changed files will do.